apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -44,4 +44,66 @@ export const contentConfig: ConfigurableContent = {
             'Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
         ].join(' '),
     },
+    termOfTheDay: {
+        '1': {
+            "01": "123",
```

Comment:
Why don't you take one of the dummy term IDs from the test data and flesh out the entire structure?

One shortcut would be to just spell out the term of the day for 31 days and then reuse this for every month. Since this config is a `ts` file, you can actually do that programatically. I can show you how if you need help.

Sorry that this approach is painful, but we just can't justify spending the time to support this in the back-end right now.